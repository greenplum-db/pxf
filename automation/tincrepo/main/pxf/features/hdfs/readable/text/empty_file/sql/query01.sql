-- @description query01 for PXF HDFS Readable emptyTextFile case

SELECT * from pxf_hdfs_small_data ORDER BY n1;