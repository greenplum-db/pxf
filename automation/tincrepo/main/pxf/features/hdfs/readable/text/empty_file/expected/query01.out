-- @description query01 for PXF HDFS Readable emptyTextFile case
SELECT * from pxf_hdfs_small_data ORDER BY n1;
 s1 | s2 | s3 | d1 | n1 | n2 | n3 | n4 | n5 | n6 | n7 | s11 | s12 | s13 | d11 | n11 | n12 | n13 | n14 | n15 | n16 | n17 
----+----+----+----+----+----+----+----+----+----+----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----+-----
(0 rows)

